torch/_higher_order_ops/wrap.py

```diff
@@ -238,3 +238,14 @@ def __call__(self, gmod, *args, **kwargs):
 
 
 tag_activation_checkpoint = TagActivationCheckpoint()
+
+
+class SpecualteSubgraphOp(HigherOrderOperator):
```

Comment:
Used to intercept the speculate_subgraph call only. So didn't implement any dispatch keys.